README.md

```diff
@@ -53,12 +53,18 @@ example of the `lcov` format [here](./lcov.md).
 cairo-coverage path/to/trace/1.json path/to/trace/2.json path/to/trace/3.json
 ```
 
-### Coverage statistics
+### Usage locally
```

Comment:
Maybe something like `viewing report` instead? I'd guess usage would be rather "how to use cairo-coverage", which we already cover in this readme

Reply:
`Viewing report` seems as a fitting name 😄